activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -255,7 +255,7 @@ def load_instances?
 
       def method_missing(method, ...)
         if fixture_sets.key?(method.name)
-          fixture(method, ...)
+          _active_record_fixture(method, ...)
```

Comment:
```suggestion
          access_fixture(method, ...)
```

Should this just be changed back to use the existing internal `access_fixture` method?